nix/format.sh

```diff
@@ -2,10 +2,6 @@
 
 export LANG="C.UTF-8"
 
-cabal format
```

Comment:
Previously we instructed contributors to run `nix run .#format` (in `CONTRIBUTING.md`) which would take care of formatting cabal files. Should we add something there about the new way of doing things?

Reply:
The most direct answer would be: Run `pre-commit` in the Nix shell. E.g. the `nixpkgs-fmt` check was only enforced by that, not by `format.sh`.

I don't feel super strongly about the pre-commit hooks, we could also move everything to `format.sh`.

Reply:
Hmm, I see. I never actually ran `pre-commit` myself. I think we can use `pre-commit`, let's just mention it in `CONTRIBUTING.md`.